src/SwedbankPay.Sdk/Payments/CardPayments/ICardPaymentsResource.cs

```diff
@@ -21,5 +28,7 @@ public interface ICardPaymentsResource
         ///// <param name="paymentExpand"></param>
         ///// <returns></returns>
         Task<CardPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
+
+        Task<CardVerify> GetVerify(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)